gateleen-scheduler/src/main/java/org/swisspush/gateleen/scheduler/Scheduler.java

```diff
@@ -137,12 +137,17 @@ private void trigger() {
 
             ExpiryCheckHandler.updateServerTimestampHeader(request);
 
-            vertx.eventBus().request(redisquesAddress, buildEnqueueOperation("scheduler-" + name, request.toJsonObject().put(QueueClient.QUEUE_TIMESTAMP, System.currentTimeMillis()).encode()),
-                    (Handler<AsyncResult<Message<JsonObject>>>) event -> {
-                        if (!OK.equals(event.result().body().getString(STATUS))) {
-                            log.error("Could not enqueue request {}", request.toJsonObject().encodePrettily());
-                        }
-                    });
+            String queueName = "scheduler-" + name;
+            JsonObject enqueOp = buildEnqueueOperation(queueName, request.toJsonObject().put(QueueClient.QUEUE_TIMESTAMP, System.currentTimeMillis()).encode());
+            vertx.eventBus().request(redisquesAddress, enqueOp, (Handler<AsyncResult<Message<JsonObject>>>) event -> {
+                if( event.failed() ){
+                    if( log.isWarnEnabled() ) log.warn("Could not enqueue request '{}' '{}'", queueName, request.getUri(), new Exception(event.cause()));
```

Comment:
Do we need to wrap the cause in an exception?

Reply:
Just imagine how the stack would look like if we don't. HINT: Our code here WILL NOT be visible in that stack.